src/ape/types/__init__.py

```diff
@@ -75,11 +80,20 @@ def __getattr__(self, item: str) -> Any:
             item (str): The name of the property.
         """
 
+        try:
+            normal_attribute = self.__getattribute__(item)
+            return normal_attribute
+        except AttributeError:
+            pass
+
         if item not in self.event_arguments:
             raise AttributeError(f"{self.__class__.__name__} has no attribute '{item}'.")
 
         return self.event_arguments[item]
 
+    def get(self, item: str, default: Optional[Any]) -> Any:
```

Comment:
why not `__getitem__`?

Reply:
`__getitem__` is also defined above! (line 72)

I like having both. Or `__contains__` or just some way of handling checking for keys